src/main/java/redis/clients/jedis/BinaryJedisCluster.java

```diff
@@ -17,11 +18,14 @@
 import java.util.Set;
 
 import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
-import redis.clients.util.JedisClusterHashTagUtil;
 
 public class BinaryJedisCluster implements BinaryJedisClusterCommands,
     MultiKeyBinaryJedisClusterCommands, JedisClusterBinaryScriptingCommands, Closeable {
 
+  public static enum Reset {
```

Comment:
This change will trigger backward incompatibility since it is exposed to `clusterReset`.